### PR TITLE
Feature liq 533 orderbook snapshots

### PR DESCRIPTION
**Pull request by pavel-khritonenko**

### CODE REVIEW

File: client/Lykke.Service.OrderBookAnalysis.Client/Lykke.Service.OrderBookAnalysis.Client.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
```
Comment (aushakov) [inline client/Lykke.Service.OrderBookAnalysis.Client/Lykke.Service.OrderBookAnalysis.Client.xml]: Remove this file

File: src/Lykke.Service.OrderBookAnalysis.AzureRepositories/OrderBookSnapshot.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Globalization;
+using System.Linq;
+using Lykke.AzureStorage.Tables;
+using Lykke.Common.ExchangeAdapter.Contracts;
+using Lykke.Service.OrderBookAnalysis.Contracts;
+
+namespace Lykke.Service.OrderBookAnalysis.AzureRepositories
+{
+    public sealed class OrderBookSnapshotEntity : AzureTableEntity, IOrderBookSnapshot
```
Comment (aushakov) [inline src/Lykke.Service.OrderBookAnalysis.AzureRepositories/OrderBookSnapshot.cs]: Rename file to "OrderBookSnapshotEntity.cs"

File: src/Lykke.Service.OrderBookAnalysis.Services/OrderBookConverter.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+using Lykke.Service.OrderBookAnalysis.AzureRepositories;
+using Lykke.Service.OrderBookAnalysis.Contracts;
+
+namespace Lykke.Service.OrderBookAnalysis.Services
+{
+    public static class OrderBookConverter
+    {
+        public static OrderBookSnapshot FromOrderBook(DateTime ts, OrderBook orderBook)
+        {
+            var mid = (orderBook.BestAskPrice + orderBook.BestAskPrice) / 2;
```
Comment (AndreyAndreev) [inline src/Lykke.Service.OrderBookAnalysis.Services/OrderBookConverter.cs]: Should it be (BestAskPrice + BestBidPrice) / 2 ?

**Review by AndreyAndreev — CHANGES_REQUESTED** [review]
One comment

File: src/Lykke.Service.OrderBookAnalysis/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            return services.BuildServiceProvider<AppSettings>(options =>
+            {
+                options.SwaggerOptions = _swaggerOptions;
+
+                options.Logs = logs =>
+                {
+                    logs.AzureTableName = "OrderBookAnalysisLog";
+                    logs.AzureTableConnectionStringResolver = settings => settings.OrderBookAnalysisService.Db.LogsConnString;
+
+                    // TODO: You could add extended logging configuration here:
```
Comment (aushakov) [inline src/Lykke.Service.OrderBookAnalysis/Startup.cs]: Remove unnecessary comments

**Review by aushakov — DISMISSED** [review]

File: src/Lykke.Service.OrderBookAnalysis.Services/OrderBookConverter.cs [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                Timestamp = ts,
+
+                CountAskOrders = orderBook.AskLevels.Count,
+                CountBidOrders = orderBook.BidLevels.Count,
+
+                SumAskVolume = orderBook.AskLevels.Sum(x => x.Value),
+                SumBidVolume = orderBook.BidLevels.Sum(x => x.Value),
+
+                Ask = orderBook.BestAskPrice,
+                Bid = orderBook.BestAskPrice,
```
Comment (aushakov) [inline src/Lykke.Service.OrderBookAnalysis.Services/OrderBookConverter.cs]: Should it be "BestBidPrice"?

**Review by aushakov — DISMISSED** [review]

File: src/Lykke.Service.OrderBookAnalysis.Services/OrderBookConverter.cs (hunk 3 above)
Comment (pavel-khritonenko) [reply src/Lykke.Service.OrderBookAnalysis.Services/OrderBookConverter.cs]: fixed + unit test has been written

**Review by AndreyAndreev — DISMISSED** [review]

**Review by AndreyAndreev — DISMISSED** [review]

**Review by AndreyAndreev — APPROVED** [review]

**Review by aushakov — APPROVED** [review]

**Pull request merged**